core/templates/components/ck-editor-helpers/ck-editor-copy-content-service.spec.ts

```diff
@@ -29,7 +29,7 @@ import { LoggerService } from 'services/contextual/logger.service';
 const generateContent = (html: string): HTMLElement => {
   const container = document.createElement('template');
   container.innerHTML = `<angular-html-bind>${html.trim()}</angular-html-bind>`;
-  // Return element inside <angular-html-bind />
+  // Return element inside <angular-html-bind />.
```

Comment:
I think you added `>` as allowed punctuation, in that case, we don't need `.` as ending punctuation here, right?

Reply:
Done!